motoman_driver/src/joint_trajectory_streamer.cpp

```diff
@@ -460,6 +460,74 @@ void MotomanJointTrajectoryStreamer::streamingThread()
         }
       }
       break;
+
+    case TransferStates::POINT_STREAMING:
```

Comment:
@shaun-edwards: does this need to be added twice? Once here and once in the `industrial_robot_client` extension part of this package?